Sources/WalletConnectPush/PushStorageIdntifiers.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
 enum PushStorageIdntifiers {
-    static let pushSubscription = "com.walletconnect.sdk.pushSbscription"
+    static let pushSubscription = "com.walletconnect.push.subscription"
```

Comment:
why this has to be changed?

Reply:
to be consistent with another sync stores namings